hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/ReturnTypes.java

```diff
@@ -103,6 +103,36 @@ private ReturnTypes() {
                 + ",string" // ledgerId
             + ")"
         + ")";
+
```

Comment:
Can we not share a lot of this with `RESPONSE_CODE_TOKEN_INFO` above? Also for the fungible and not versions below.